src/components/TimeCardPage/SubmitCard.tsx

```diff
@@ -8,39 +8,57 @@ import Button from 'react-bootstrap/Button';
 
 export default function SubmitCard(props) {
 
-    const [submitted, setSubmitted] = useState(false); 
-    const [submitDate, setSubmitDate] = useState(null); 
+    const CardState = {
+        Rejected: "Rejected",
+        InReviewEmployer: "In Review - Employer",
+        InReviewBreaktime: "In Review - Breaktime",
+        Completed: "Completed"
+    }
+
+    const [submitted, setSubmitted] = useState(false);
+    const [submitDate, setSubmitDate] = useState(null);
+    const [state, setState] = useState(CardState.Rejected);
+    const [rejected, setRejected] = useState(false);
 
     useEffect(() => {
         //TODO - API Call to determine if the table has been submitted or not.
-        //Will set submitted? here and also submitDate if it was submitted to grab the date 
-        
-    },[])
+        //Will set submitted? here and also submitDate if it was submitted to grab the date     
+    }, [props.comment])
 
     const submitAction = () => {
-        setSubmitted(!submitted); 
-        const currentTime = new Date(); 
-        setSubmitDate(currentTime.toString()); 
+        setSubmitted(!submitted);
+        const currentTime = new Date();
```

Comment:
I think our new convention is to use `moment`, so this would be `const currentTime = moment();`
@kennedyry

Reply:
Why use `moment`? Its a large dependency so if we can avoid it we should

Reply:
We use moment so it's a lot easier to manipulate dates and times for the timetable/timesheet. We could use the default date object but we would have to implement a lot of the things moment has done so just leads to reinventing the wheel somewhat.